PeerioTechnologies/peerio-desktop
# feat: jumboji

#### Relevant info and issue/PR links

https://app.clubhouse.io/peerio/story/10386/desktop-jumboji


#### Testing instructions

All emoji images were bumped up to 64px to allow for the larger sprites; shouldn't affect things but please check around (on different platforms if possible) just in case. (It keeps them from being blurry when they grow-on-hover anyway.)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
